dev-util/phpstorm/phpstorm-9.0.2.141.2462.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+inherit eutils versionator
+
+SLOT="$(get_major_version)"
+MY_PV="$(get_version_component_range 4-5)"
+MY_PN="PhpStorm"
+MY_PN2="webide"
+
+DESCRIPTION="hpStorm’s smart code editor provides excellent support for PHP (including the latest language versions and frameworks), HTML, JavaScript, CSS, Sass, Less, CoffeeScript, and many other languages. Enjoy coding with intelligent context-aware code completion, error detection, and on-the-fly code inspections & fixes."
+HOMEPAGE="http://www.jetbrains.com/phpstorm"
+SRC_URI="http://download.jetbrains.com/${MY_PN2}/${MY_PN}-$(get_version_component_range 1-3).tar.gz"
+
+LICENSE="PhpStorm_license PhpStorm_Academic_license PhpStorm_Classroom_license PhpStorm_OpenSource_license PhpStorm_personal_license"
```

Comment:
Not sure if some of them shouldn't use `|| ()`. Do you need to read all of them to use this? ;-P